src/qmflows/parsers/orca_parser.py

```diff
@@ -16,14 +16,15 @@
 from .parser import (floatNumber, natural, parse_file, parse_section, skipLine,
                      skipSupress, string_array_to_molecule, try_search_pattern)
 from .xyzParser import manyXYZ
+from ..type_hints import PathLike
 
 Vector = np.ndarray
 Matrix = np.ndarray
 
 vectorize_float = np.vectorize(float)
 
 
-def parse_molecule(file_name, mol=None):
+def parse_molecule(file_name: PathLike, mol: Molecule = None) -> Molecule:
```

Comment:
The  ``mol`` annotation should be ``Optional[Molecule]`` (*i.e.* ``None`` or a ``Molecule`` instance)

Reply:
indeed